packages/logger/tslint.json

```diff
@@ -42,7 +42,7 @@
     "no-empty": true,
     "no-floating-promises": true,
     "no-for-in-array": true,
-    "no-implicit-dependencies": true,
+    "no-implicit-dependencies": [true, "dev"],
```

Comment:
This one could be helpful for other packages if you'd like to write tests in TypeScript. (Currently all the existing tests in other packages are written in JS)

Reply:
I realized other projects never use any transpilers and also found the following rule in `node-slack-interactive-messages`:

> Test code should be written in syntax that runs on the oldest supported Node.js version, without transpiling. This ensures that backwards compatibility is tested and the APIs look reasonable in versions of Node.js that do not support the most modern syntax.

https://github.com/slackapi/node-slack-interactive-messages/blob/master/.github/maintainers_guide.md#testing

I've updated this pull request not to use TypeScript this time.